apple/Sources/FerrostarSwiftUI/Theme/RoadNameViewTheme.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+public protocol RoadNameViewTheme: Equatable {
+    /// The color for the road name label.
+    var textColor: Color { get }
+
+    /// The font for the road name label.
+    var textFont: Font { get }
+
+    /// The color of the background.
+    var backgroundColor: Color { get }
+
+    /// The color of the border around the view.
+    var borderColor: Color { get }
+}
+
+public struct DefaultRoadNameViewTheme: RoadNameViewTheme {
+    public var textColor: Color = .init(.white)
+    public var textFont: Font = .callout.bold()
+    public var backgroundColor: Color = .init(.systemBlue)
+    public var borderColor: Color = .white
```

Comment:
Same as android. Let's build something like: 

```swift
public final class FerrostarColors {
    public static let roadColor = Color.something
    public static let roadCasingColor = Color.something
}
```

If we do that, we might even be able to easily match the route colors to the this object. Since we have the theme objects, they can offer the customization and this can just be a one stop for validating and modifying colors throughout the system.

Reply:
Yeah, good call. I think you're a bit better positioned to extract all this, so I've put some notes in #315.